### PR TITLE
Prod stable summit release

### PR DESCRIPTION
**Pull request by ryelo**
DO NOT MERGE UNTIL MONDAY

### CODE REVIEW

**Comment by ryelo** [comment]
Added module to apps frontend #551

File: main.yml [hunk 1]
```diff
@@ -168,8 +218,11 @@ compliance:
   api:
     versions:
       - v1
+    isBeta: true
```
Comment (karelhala) [inline main.yml]: I don't think this is correct.
```suggestion
```

File: main.yml [hunk 2]
```diff
@@ -193,8 +246,10 @@ cost-management:
...
```
Comment (karelhala) [inline main.yml]:
```suggestion
```

File: main.yml [hunk 3]
```diff
@@ -375,82 +462,134 @@ inventory:
...
+      - id: cost-management
+
   top_level: true
 
 patch:
   title: Patch
   api:
     versions:
       - v1
+    isBeta: true
```
Comment (karelhala) [inline main.yml]:
```suggestion
```

File: main.yml [hunk 4]
```diff
@@ -561,17 +705,24 @@ settings:
...
+        title: Documentation
+        navigate: https://access.redhat.com/documentation/red_hat_cloud_platform/
 
 sources:
   title: Discovered Inventory
   api:
     versions:
       - v1
-  channel: '#insights_topology_svc'
+    isBeta: true
```
Comment (karelhala) [inline main.yml]:
```suggestion
```

**Review by karelhala — APPROVED** [review]
There's a lot of beta flags for APIs, if we'd keep the values in there they won't show up in API docs.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**